### DISCUSSION TITLE
Request for more hours on GitHub pro plan.

### DISCUSSION THREAD

**Discussion by manuthecoder** [community/community#39295] [Codespaces]
### Product Feedback

Hi,
I am currently a GitHub codespaces pro user, and I find that 180 hours isn't much of a difference from 120 hours given in the free plan. 

It makes sense to provide more hours, especially since pro users pay for this, and looking at it from a business perspective, this would make the pro pack much more valuable! I, personally, heavily rely on Codespaces for the development of my app.

In my opinion, a reasonable amount of hours for the Codespaces pro plan would be: **240-300**

EDIT: Yes, I am a pro user, and this is why I am submitting this post.

**Comment by erikaellr** [comment]
Hi @manuthecoder, we're thrilled that you're using Codespaces so much! We are always working to make GitHub, and Codespaces, as awesome as possible, and while we can't guarantee that every bit of feedback sees action, we are always listening, and I have passed on your feedback accordingly.

**Reply by manuthecoder** [reply]
Thank you for conveying my feedback to your team! I would like to be informed of any further updates.

**Comment by d9j** [comment]
even on free plan 120 hours. is hard to justify usage outside  of hobby projects
